contrib/bcachefs-tools/template.py

```diff
@@ -47,3 +46,10 @@ def init_build(self):
     ]
     # apply our rust stuff
     self.env.update(cargo.get_environment(self))
+
+
+def init_install(self):
+    self.make_install_args += [
```

Comment:
self.profile() works from toplevel, no need for an init_ func

Reply:
doh, forgot that. thanks!